runtime/tests/Test_LightningCoreQIS.cpp

```diff
@@ -261,12 +261,22 @@ TEST_CASE("Test copy to strided array", "[qir_lightning_core]")
     size_t buffer_len = data.size();
     double *buffer = data.data();
     MemRefT<double, 2> src = {buffer, buffer, 0, {buffer_len / 2, 2}, {2, 1}};
+    MemRefView<double, 2> src_view(&src, buffer_len);
+    CHECK(src_view(0, 0) == buffer[0]);
+    CHECK(src_view(0, 1) == buffer[1]);
+    CHECK(src_view(1, 0) == buffer[2]);
+    CHECK(src_view(1, 1) == buffer[3]);
+    CHECK(src_view(2, 0) == buffer[4]);
+    CHECK(src_view(2, 1) == buffer[5]);
+    CHECK(src_view(3, 0) == buffer[6]);
+    CHECK(src_view(3, 1) == buffer[7]);
 
     size_t buffer_strided_len = buffer_len * 2;
     double *buffer_strided = new double[buffer_strided_len];
     MemRefT<double, 2> dst = {
         buffer_strided, buffer_strided, 0, {buffer_strided_len / 2, 2}, {4, 2}};
-    memref_copy<double, 2>(&dst, &src, buffer_len * sizeof(double));
+    MemRefView<double, 2> dst_view(&dst, buffer_strided_len);
```

Comment:
I think you are modifying this test to check something else. This test explicitly tests to make sure that copies work on strided arrays. It looks like the clone function goes through a different path from `memref_copy`.

Reply:
That is why code coverage is issuing a warning [here](https://github.com/PennyLaneAI/catalyst/commit/a141fd5a60e3f2d032cdc7c470ef108520e5e686#diff-ac5991ca62622adc73a1bf5919adddab6b5cbaa79113056deb35eb0fc8dff413R198)

Reply:
The warning makes sense as the `memref_copy` is no longer used in measurements' tests and it will be removed after fixing the gradient signature. `MemRefView<T, R>::clone` is the new name for `memref_copy`  and is tested in this test function.